server/server.go

```diff
@@ -98,11 +98,21 @@ func (s *server) ServeSearch(ctx context.Context, w http.ResponseWriter, r *http
 		bk.I.Unlock()
 	}
 
+	link_configs := make([]map[string]string, len(s.config.LinkConfigs))
+	for idx, link_config := range s.config.LinkConfigs {
+		link_configs[idx] = map[string]string{
```

Comment:
Can we just pass the `s.config.LinkConfigs` object directly instead of destructuring it and restructuring it into an identically-shaped object with different names? If we switch to using `_` in the JSON it'll even still be easy-to-access as a Javascript object.